microservices/kubeApi/routers/routes.py

```diff
@@ -194,41 +198,38 @@ async def verify_and_create_routes(namespace: str, request: Request):
             for route in insert_batch:
                 overrides = {}
                 if 'sessionCookieEnabled' in route and route['sessionCookieEnabled']:
-                    overrides['aps.route.session.cookie.enabled'] = [ route['host'] ]
+                    overrides['aps.route.session.cookie.enabled'] = [route['host']]
+                
                 route_count = prepare_apply_routes(namespace, route['selectTag'], [
                                                    route['host']], source_folder, route["dataPlane"], ns_template_version, overrides)
+                
                 logger.debug("[%s] - Prepared %d routes" % (namespace, route_count))
                 apply_routes(source_folder)
                 logger.debug("[%s] - Applied %d routes" % (namespace, route_count))
+
+                inserted_count += route_count
     except Exception as ex:
         traceback.print_exc()
         logger.error("Error creating routes. %s" % (ex))
         raise HTTPException(status_code=400, detail="Error creating routes. %s" % (ex))
-    except SystemExit as ex:
```

Comment:
Are you sure it is OK to remove the raising of the SystemExit exception?  I think I had to include this otherwise the SIGTERM signal is ignored if the process is in the middle of that code block.  So could take longer to terminate the Pod.

Reply:
My bad - I had removed earlier when trying to resolve errors that came up when deleting a route that doesn't exist (until I saw and modeled off `microservices/kubeApi/tests/routers/test_del_route.py`) and should have restored them.

More generally, I should have reviewed all my changes in the commit. Thanks for the catch